src/codeu/chat/client/simplegui/Conversation.java

```diff
@@ -0,0 +1,111 @@
+package codeu.chat.client.simplegui;
+
+
+import java.awt.BorderLayout;
+import java.awt.Button;
+import java.awt.Container;
+import java.awt.FlowLayout;
+import java.awt.GridBagConstraints;
+import java.awt.GridBagLayout;
+import java.awt.event.MouseAdapter;
+import java.awt.event.MouseEvent;
+
+import javax.swing.BoxLayout;
+import javax.swing.JButton;
+import javax.swing.JFrame;
+import javax.swing.JLabel;
+import javax.swing.JPanel;
+import javax.swing.JTextArea;
+import javax.swing.JTextField;
+
+/**
+ * @author  Troy Jones
+ * @date    5/21/17
+ * @brief   Contains all GUI necessary to execute the frame to 
+ *          enable chat.
+ */
+@SuppressWarnings("serial")
+public class Conversation extends JFrame {
+
+    // constructor
+    public Conversation() {
+
+        // Used to see if user has clicked inside the editable
+        //  textarea. If true, the default text will clear. If false, the 
+        //  text inside stays.
+        boolean isFirstClick = true;
+
+        JFrame frame = new JFrame();
+
+        // Main container that components will go inside. Will then be appended
+        //  to frame.
```

Comment:
same here, but this can go above?

also kinda super nit - if you capitalize first word and end with a period, please do that on all the comments =)